src/Tribe/Customizer/Controls/Heading.php

```diff
@@ -44,7 +44,7 @@ class Heading extends Control {
 	 */
 	public function render_content() {
 		?>
-		<h4 style="font-size: 16px; font-weight: normal; line-height: 1.75; margin-top: 0; margin-bottom: 0px;">
+		<h4 style="font-size: 20px; font-weight: normal; line-height: 1.75; margin-top: 0; margin-bottom: 0px;">
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: Line exceeds maximum limit of 100 characters; contains 112 characters
* :no_entry_sign: PHP files must only contain PHP code

<sub>hash: 168f7e5b433ab0f95e18484245ddaff9</sub>